config/packages/fos_http_cache.yaml

```diff
@@ -1,7 +1,7 @@
 fos_http_cache:
     proxy_client:
         symfony:
-            use_kernel_dispatcher: true
+            use_kernel_dispatcher: true # disable this when varnish is used
```

Comment:
@alexander-schranz Should that be always `true` on dev and just be changed to `false` on prod, if varnish is used? Then I would keep it like it was and just close this pr